apps/cowswap-frontend/src/modules/limitOrders/services/tradeFlow/index.ts

```diff
@@ -71,11 +70,7 @@ export async function tradeFlow(
     })
 
     if (postOrderParams.appData.fullAppData.includes(PERMIT_SIGNER.address)) {
-      // report this to sentry if we ever use the default signer in the permit
-      Sentry.captureException('User signed the permit using PERMIT_SIGNER instead of their account', {
-        tags: { errorType: 'permitWithDefaultSigner' },
-        contexts: { params: { account } },
-      })
+      reportPermitWithDefaultSigner(postOrderParams)
```

Comment:
It was only set on limit.
Extracted to a helper and expanded to log all order params, not only the owner address.